src/journal/journald-kmsg.c

```diff
@@ -385,6 +385,40 @@ static int dispatch_dev_kmsg(sd_event_source *es, int fd, uint32_t revents, void
         return server_read_dev_kmsg(s);
 }
 
+int server_reload_dev_kmsg(Server *s) {
+        int r;
+
+        assert(s);
+
+        mode_t mode = O_CLOEXEC|O_NONBLOCK|O_NOCTTY|
+                (s->read_kmsg ? O_RDWR : O_WRONLY);
```

Comment:
this line is used at two locations now. and it's not entirely trivial, might get out of sync. can you add a quick helper for this:

```c
mode_t server_kmsg_mode(Server *s) {
        …
```

and then use that at both places